tests/rptest/archival/s3_client.py

```diff
@@ -182,13 +182,14 @@ def create_bucket(self, name):
         def bucket_is_listable():
             try:
                 self._cli.list_objects_v2(Bucket=name)
-            except:
-                self.logger.warning(f"Listing {name} failed after creation")
+            except Exception as e:
+                self.logger.warning(
+                    f"Listing {name} failed after creation: {e}")
```

Comment:
nit: i think you can also do `self.logger.exception` and it will automatically log the current exception.

Reply:
TIL 🙇‍♂️